README.md

```diff
@@ -1,21 +1,22 @@
-[![Sourcecode License](https://img.shields.io/pypi/l/pyTooling?logo=GitHub&label=code%20license)](LICENSE.md)
-[![GitHub tag (latest SemVer incl. pre-release)](https://img.shields.io/github/v/tag/pyTooling/pyTooling?logo=GitHub&include_prereleases)](https://GitHub.com/pyTooling/pyTooling/tags)
-[![GitHub release (latest SemVer incl. including pre-releases)](https://img.shields.io/github/v/release/pyTooling/pyTooling?logo=GitHub&include_prereleases)](https://GitHub.com/pyTooling/pyTooling/releases/latest)
-[![GitHub release date](https://img.shields.io/github/release-date/pyTooling/pyTooling?logo=GitHub)](https://GitHub.com/pyTooling/pyTooling/releases)
-[![Dependents (via libraries.io)](https://img.shields.io/librariesio/dependents/pypi/pyTooling?logo=librariesdotio)](https://GitHub.com/pyTooling/pyTooling/network/dependents)  
-[![GitHub Workflow - Build and Test Status](https://img.shields.io/github/workflow/status/pyTooling/pyTooling/Unit%20Testing,%20Coverage%20Collection,%20Package,%20Release,%20Documentation%20and%20Publish?label=Pipeline&logo=GitHub%20Actions&logoColor=FFFFFF)](https://GitHub.com/pyTooling/pyTooling/actions/workflows/Pipeline.yml)
-[![Codacy - Quality](https://img.shields.io/codacy/grade/8dc5205ba8b24e008f2287759096e181?logo=Codacy)](https://www.codacy.com/gh/pyTooling/pyTooling)
-[![Codacy - Coverage](https://img.shields.io/codacy/coverage/8dc5205ba8b24e008f2287759096e181?logo=Codacy)](https://www.codacy.com/gh/pyTooling/pyTooling)
-[![Codecov - Branch Coverage](https://img.shields.io/codecov/c/github/pyTooling/pyTooling?logo=Codecov)](https://codecov.io/gh/pyTooling/pyTooling)
-[![Libraries.io SourceRank](https://img.shields.io/librariesio/sourcerank/pypi/pyTooling?logo=librariesdotio)](https://libraries.io/github/pyTooling/pyTooling/sourcerank)  
-[![PyPI](https://img.shields.io/pypi/v/pyTooling?logo=PyPI&logoColor=FBE072)](https://pypi.org/project/pyTooling/)
-![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pyTooling?logo=PyPI&logoColor=FBE072)
-![PyPI - Status](https://img.shields.io/pypi/status/pyTooling?logo=PyPI&logoColor=FBE072)
-[![Libraries.io status for latest release](https://img.shields.io/librariesio/release/pypi/pyTooling?logo=librariesdotio)](https://libraries.io/github/pyTooling/pyTooling)
-[![Requires.io](https://img.shields.io/requires/github/pyTooling/pyTooling)](https://requires.io/github/pyTooling/pyTooling/requirements/?branch=main)  
-[![Documentation License](https://img.shields.io/badge/doc%20license-CC--BY%204.0-green?logo=readthedocs)](doc/Doc-License.rst)
-[![Documentation - Read Now!](https://img.shields.io/badge/doc-read%20now%20%E2%9E%9A-blueviolet?logo=readthedocs)](https://pyTooling.GitHub.io/pyTooling)
+[![Sourcecode on GitHub](https://img.shields.io/badge/pyTooling-pyTooling-8cd665.svg?longCache=true&style=flat-square&logo=github&longCache=true&logo=GitHub&labelColor=63bf7f)](https://GitHub.com/pyTooling/pyTooling)
```

Comment:
I'm not sure if the 2 green colors fit well to each other in the same shield (first shield).

![image](https://user-images.githubusercontent.com/956109/149711439-89988e30-c1f0-47a7-b10e-b0db3e446941.png)

Reply:
Updated to use the default gray on the left side.